### PR TITLE
Allow piping to cvlc or ffmpeg

### PR DESCRIPTION
**Pull request by maingoh**
It was already possible to pipe the frames on stdout as RGB. This PR intends to generalize the output color space so that we can choose which one to output. Since internally our frames are in BGR format, we use it now as default. Added `--output_color_space` to change it. 
See README for me example on how to use it.

### CODE REVIEW

**Review by alemoreau — APPROVED** [review]
Great ! For testing the stdout output, maybe one day we could refactor tests to run the commands in subprocesses, capture the output and compare with a file from another command.

**Comment by maingoh** [comment]
Yes I was thinking about something like this, maybe just an e2e test with bash commands

File: README.md [hunk 1]
```diff
@@ -64,6 +64,47 @@ pip uninstall opencv-python-headless
...
+In both case you need to tell `ffmpeg` or `cvlc` about the resolution, framerate and color space of the input stream.
+You can use `ffprobe` or `mediainfo` to get the resolution and framerate of your input video.
+The color space (chroma) does not depend on your input video but on our CLI which by default output BGR color space.
+
+Again, make sure you can legally use the codec specified in the command.
+
+#### Example using `ffmpeg`
+
+```bash
+deepo platform model noop -i $input_video_path -o stdout | ffmpeg -f rawvideo -pixel_format bgr24 -video_size 1280x720 -framerate 15 -i - -c:v $codec $output_video_path
```
Comment (romyli) [inline README.md:97]:
```suggestion
deepo platform model draw -i $input_video_path -o stdout -r $model_id | ffmpeg -f rawvideo -pixel_format bgr24 -video_size 1280x720 -framerate 15 -i - -c:v $codec $output_video_path
```

File: README.md [hunk 2]
```diff
@@ -64,6 +64,47 @@ pip uninstall opencv-python-headless
...
+```
+
+#### Example using `cvlc`
+
+BGR color space is not supported by `cvlc`, so we have to convert the stream to `RGB`.
+
+```bash
+deepo platform model noop -i $input_video_path -o stdout --output_color_space RGB | cvlc --demux=rawvideo --rawvid-fps=15 --rawvid-width=1280 --rawvid-height=720 --rawvid-chroma=RV24 - --sout "#transcode{vcodec=$codec}:std{access=file,dst=$output_video_path}" vlc://quit
```
Comment (romyli) [inline README.md:105]:
```suggestion
deepo platform model draw -i $input_video_path -o stdout -r $model_id --output_color_space RGB | cvlc --demux=rawvideo --rawvid-fps=15 --rawvid-width=1280 --rawvid-height=720 --rawvid-chroma=RV24 - --sout "#transcode{vcodec=$codec}:std{access=file,dst=$output_video_path}" vlc://quit
```

**Review by romyli — APPROVED** [review]

**Review by alemoreau — APPROVED** [review]

**Pull request merged**